mast/integrations/utils.py

```diff
@@ -0,0 +1,386 @@
+import json
+import requests
+from flask_login import current_user
+from datetime import time, datetime
+from mast.queries import Queries
+from mast.models import  Activity, ActivityType, Sex, User
+from mast.tools.points import Points
+
+# TODO: remove
+import logging
+
+strava_logger = logging.getLogger('STRAVA')
+
+# TODO: change these in the production since they are already public
+STRAVA_CLIENT_ID = 61623
+STRAVA_CLIENT_SECRET = '71af3bcd89c9a583607db1a383e36f8c1cf6790a'
+STRAVA_SCOPE = ['activity:read']
+
+
+class ExpiredAccessToken(Exception):
```

Comment:
This is on me, it can be removed I believe I have come up with a better way to deal with token expiration.